template/next-ts-chakra-ui/cypress/support/index.ts

```diff
@@ -1,2 +1,3 @@
+/* eslint-disable import/no-extraneous-dependencies */
```

Comment:
It might be better for us to update this override instead to add the `cypress` folder:

<img width="545" alt="image" src="https://user-images.githubusercontent.com/890171/122308939-b8d17480-cf50-11eb-86ca-90cbe9d3a009.png">

Reply:
yeah probably makes sense